index.js

```diff
@@ -37,9 +37,9 @@ var multipartser = function multipartser () {
     if ( splits.length < 3 ) { return; }
 
     // we found at least one part
-    // the first element should be empty string indicating that 
+    // the first element should be empty string indicating that
     // boundary was at the beginning of the body
-    if ( splits[ 0 ] != '' ) {
+    if ( !splits[ 0 ].match(/^\s+$/) ) {
```

Comment:
I think the regex should be `/^\s+\r\n$/` because `--` must come at the beginning of the line according to [RFC2046](http://www.ietf.org/rfc/rfc2046.txt);
> The boundary delimiter MUST occur at the beginning of a line, i.e.,
   following a CRLF, and the initial CRLF is considered to be attached
   to the boundary delimiter line rather than part of the preceding
   part.